src/map/script.cpp

```diff
@@ -24128,6 +24128,12 @@ BUILDIN_FUNC(minmax){
 	return SCRIPT_CMD_SUCCESS;
 }
 
+BUILDIN_FUNC(cap_value)
+{
+	script_pushint64(st, (int64)cap_value(script_getnum64(st, 2), script_getnum64(st, 3), script_getnum64(st, 4)));
```

Comment:
```suggestion
	script_pushint64(st, cap_value(script_getnum64(st, 2), script_getnum64(st, 3), script_getnum64(st, 4)));
```
Should not be necessary to cast here. You call the funciton with all variables being int64, so the macro should return an int64 as well. See:
```cpp
#define cap_value(a, min, max) (((a) >= (max)) ? (max) : ((a) <= (min)) ? (min) : (a))
```